moscar.go

```diff
@@ -1,124 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"github.com/micromdm/squirrel/munki/datastore"
-	"github.com/micromdm/squirrel/munki/models"
-	"html/template"
-	"io"
-	"log"
-	"net/http"
-	"time"
-)
-
-const STATIC_URL string = "/static/"
```

Comment:
In Go, the preferred style is `camelCase` and linters will complain if you use underscores for your variables.
Additionally, const/var/functions/etc which are capitalized are exported, meaning they are part of your public API. Unless there's an explicit reason for something to be exported, I'd lowercase it. 

Following the above advice, I'd change these to be 

```
const (
	staticURL  string = "/static/"
	staticRoot string = "static/"
	repoPath   string = "/Users/Shared/repo"
)
```